### PR TITLE
Add org.munadi.Munadi

### PR DESCRIPTION
**Pull request by hakpw**
After waiting more than 7 months, it seems we aren't getting autorun figured out by Flatpack team, nor Flathub allowing access to `autorun` dir, therefore I decided to just do it the open source way, i.e. ship it half baked!

Welldone everyone!

### CODE REVIEW

**Comment by hakpw** [comment]
bot, build org.munadi.Munadi

**Comment by hakpw** [comment]
bot, build org.munadi.Munadi

**Comment by barthalion** [comment]
Eh, as discussed previously, users still can do it on their own via autostart feature of desktop environment, flatpak just doesn't expose it to the app itself.

File: org.munadi.Munadi.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "app-id": "org.munadi.Munadi",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
```
Comment (barthalion) [inline org.munadi.Munadi.json]: Can you try 5.12?

**Comment by hakpw** [comment]
bot, build org.munadi.Munadi

**Comment by hakpw** [comment]
> Eh, as discussed previously, users still can do it on their own via autostart feature of desktop environment, flatpak just doesn't expose it to the app itself.

I know, but its a question of usability, not wether it's possible or not. Never mind, we'll get there one day :-)

File: org.munadi.Munadi.json (hunk 1 above)
Comment (hakpw) [reply org.munadi.Munadi.json]: I did and it failed! Just checked, new Qt JS engine broke it, I've updated the source, testing it now ....

Comment (hakpw) [reply org.munadi.Munadi.json]: @barthalion All done, tested fully working, thanks.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.munadi.Munadi

Thanks!

**Comment by hakpw** [comment]
@barthalion Thanks for your time, much appreciated :-)

**Pull request closed without merging**